GOVERNANCE.md

```diff
@@ -288,7 +287,9 @@ Contributor roles are granted for as long as the person wishes to engage with th
 
 **Alumni** is a special designation and role for any person who was once an active maintainer (L2 or above) but is now no longer actively involved. By retiring and joining Alumni you trade-in your current set of roles, privileges, and responsibilities for a new, special Alumni role (which comes with its own set of Privileges, as described above).
 
-As a Maintainer (L2 or above) you can retire your role at any time by pinging the project Steward and requesting Alumni status. You can initiate this action yourself if you know ahead-of-time that you need to step away from the project. Or, if you have gone several months without interacting with the Astro community, the project Steward may actively reach out to you to discuss retiring as a way to make room for new contributors.
+As a Maintainer (L2 or above) you can retire your role at any time by pinging the project Steward and requesting Alumni status. You can initiate this action yourself if you know ahead-of-time that you need to step away from the project.
+
+If you are not actively contributing to Astro’s repositories or the Astro community for longer than six months, the project Steward may proactively retire your role, designate you as a project alumnus, and notify you of the change.
```

Comment:
> and notify you of the change.

Can we leave this vague? In the past, I've tried nudging inactive contributors to alumni, and heard back "oh no I'm totally about to be active again, I was just $REASON" and they post a bit for a day or two and then go back to being inactive.

I do think a notification is good practice but as the person who will be implementing this I'd personally like the freedom to decide if the proactive ping is warrented. Ex: do it the first time, but not the second.

Not the end of the world, just my 2 cents.

Reply:
Yes, I think what's also important here is that we are never forced to wait to hear back from someone. Not responding in a reasonable amount of time is probably a clear sign they are not active, and that's exactly who we should feel comfortable alumni'ing, I think!

Reply:
I'm not sure I understand what you're suggesting. Can you please suggest a different phrasing?

Reply:
Me? This is what I'm suggesting:

> If you are not actively contributing to Astro’s repositories or the Astro community for longer than six months, the project Steward may proactively retire your role and designate you as a project alumnus ~~, and notify you of the change.~~

This leaves the notification out of governance. In practice I would do this often, but not always.

Reply:
The structure of the sentence does make it confusing! I thought of each action here as independently optional, but @FredKSchott is equally valid in interpreting the action as a single group that is itself optional.


On principle, if someone has reached L2+, I feel like they’ve earned a heads up on any status change. But I understand there might be situations where that’s not desirable. IMO the status change should at least be shared with the rest of core?